tests/test_dump_load.py

```diff
@@ -1,15 +1,15 @@
 import pytest
-import yaml
```

Comment:
I'd just do `import yaml_ft as yaml` so you can ovoid other changes in this file